README.md

```diff
@@ -6,20 +6,31 @@ Typographical styles for FT branded sites - font families, weight, colors, sizes
 
 ## Overview
 
-This module provides styles for Headings, Titles, Leads, Metadata, and body content.
+This module provides styles for Headings, Titles, Leads and body content.
 
-Body content styles come in two categories:
+### Font system
 
-* **general**, used for informational pages, forms, application user interfaces etc.
-* **article**, used for editorial content.
+We’re using a matrix of font variants in order to standardize typography across the site. This provides a common language and helps to avoid inconsistencies.
 
-### Web fonts
+#### Sans
+[![Font system sans](https://raw.githubusercontent.com/Financial-Times/o-typography/next-type/img/matrix-sans.png)](https://raw.githubusercontent.com/Financial-Times/o-typography/next-type/img/matrix-sans.png)
 
-o-typography **does not load web fonts**, products should load web fonts themselves.
+#### Serif
+[![Font system serif](https://raw.githubusercontent.com/Financial-Times/o-typography/next-type/img/matrix-serif.png)](https://raw.githubusercontent.com/Financial-Times/o-typography/next-type/img/matrix-serif.png)
 
-**Load FT's custom web fonts using [o-fonts](https://github.com/financial-times/o-fonts).**
+### Recommended setup
 
-## Usage
+```
+html {
+	font-family: $o-typography-sans;
+	-ms-text-size-adjust: 100%;
+	-webkit-text-size-adjust: 100%;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+	text-rendering: optimizeLegibility;
+	-moz-font-feature-settings: "liga" on;
```

Comment:
might be good to say why and are all these really necessary? Might make people think it's hard to get started